container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/user_services_functions/persistent_directories.go

```diff
@@ -12,14 +12,12 @@ import (
 )
 
 type kubernetesVolumeWithClaim struct {
```

Comment:
Change Default size to 1g as thats minimum for. the dynamic provisioning drivers

Reply:
good